### ISSUE TITLE
System.SObjectException: Invalid field CurrencyIsoCode for Contact

### ISSUE THREAD

**Issue by up2go-sschwarz** [afawcett/apex-sobjectdataloader#21]
These four tests will fail in orgs where **Multiple Currencies** are **NOT** enabled.

- userFieldWhiteListTest
- userChildWhiteListTest
- userFieldWhiteListTestWithPackageBlaclist
- userChildWhiteListTestWithPackageBlaclist

**Comment by afawcett** [comment]
Thanks @up2go-sschwarz. We do need to support this in a late bound way to support both org types.

**Comment by PimdeVos** [comment]
@afawcett I found an quick solution to this one (experienced the same problem in an org today).
in those test methods there is an initialize con.put('CurrencyIsoCode','USD'); just encapsulate it with the if(UserInfo.IsMultiCurrencyOrganization()) block thats also used @ line 646 in the testOmitCommonFields method. this solved the problem for me.

Appart from this I also removed CurrencyIsoCode references in the json of processUnorderedRecordsinJsonTest which is a bit of a quick and dirty fix I admit, that perhaps also needs to be encapsulated with the above if check generated 2 different jsons.
